pkgs/development/libraries/wxwidgets/wxGTK31.nix

```
@@ -17,20 +17,21 @@
 , compat28 ? false
 , compat30 ? true
 , unicode ? true
-, withGtk2 ? true
+, withGtk2 ? (!stdenv.isDarwin)
 , withMesa ? lib.elem stdenv.hostPlatform.system lib.platforms.mesaPlatforms
-, withWebKit ? false, webkitgtk
+, withWebKit ? stdenv.isDarwin
+, webkitgtk
 , darwin
```

```suggestion
, 
```
Please input the frameworks individually.

---

Gladly! I just took the existing style and was wondering why it's only having darwin.